pkgs/data/icons/gruvbox-dark-icons-gtk/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "gruvbox-dark-icons-gtk";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "jmattheis";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1fks2rrrb62ybzn8gqan5swcgksrb579vk37bx4xpwkc552dz2z2";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/icons/
+    rm README.md
+    cp -r ${src} $out/share/icons/${pname}
+  '';
+
+  dontDropIconThemeCache = true;
+
+  meta = with stdenv.lib; {
+    description = "Gruvbox icons for GTK based desktop environments";
+    homepage = "https://github.com/jmattheis/gruvbox-dark-gtk";
+    license = licenses.gpl3;
```

Comment:
gpl3Plus or gpl3Only?

Reply:
gpl3Only, I think? Is licenses.gpl3 is deprecated, or is licenses.gpl3Olny simply preferred?
Link: https://github.com/jmattheis/gruvbox-dark-gtk/blob/master/LICENSE

Reply:
`licenses.gpl3` is deprecated indeed.

https://github.com/NixOS/nixpkgs/blob/bc91fe1935b7ac1699e166bd99863a0ae6b6efde/lib/licenses.nix#L850-L853